docs/getting-started/quickstart.md

```diff
@@ -306,19 +312,22 @@ cd <app-name>
 
 ### 3. Deploy!
 
-Run the following command to deploy your app to DBOS Cloud:
+Install the DBOS Cloud CLI:
+
 ```
-npx dbos-cloud app deploy
+npm i -g @dbos-inc/dbos-cloud@latest
 ```
 
-This command first prompts you to login, or register if this is your first time. Then, it prompts you to provision a database instance. Finally, it uploads your code to DBOS Cloud and deploys your app. In about a minute, it should succeed and print `Successfully deployed <app-name>! Access your application at <URL>`.
-
-To see that your app is working, visit `<URL>` in your browser. For example, if your username is `mike` and your app name is `hello`, you would visit:
+Then run the following command to deploy your app to DBOS Cloud:
 ```
-https://mike-hello.cloud.dbos.dev/
+dbos-cloud app deploy
 ```
 
-Congratulations, you've successfully deployed your first app to DBOS Cloud! You can see your deployed app in the [cloud console](https://console.dbos.dev/), or in the CLI by running `npx dbos-cloud app list`.
+This command first prompts you to login, or register if this is your first time. Then, it prompts you to provision a database instance. Finally, it uploads your code to DBOS Cloud and deploys your app. In under a minute, it should succeed and print `Successfully deployed <app-name>! Access your application at <URL>`.
```

Comment:
I see you may have a strong preference here :)
May I at least suggest `In less than a minute, it should succeed`. The phrasing `In under` feels off.

Reply:
Fixed :)